src/Screens/Explore/components/DiveSiteWithoutNavObject.tsx

```diff
@@ -0,0 +1,367 @@
+import React from 'react';
```

Comment:
i don't think duplicating all this code feels like the correct way to handle this logic. at the very least, we need to them break the whole screen up into smaller components that are imported into dive site with/without nav object, but otherwise it will get really hard to maintain these separate components as we make other design changes

Reply:
Yes (about breaking the entire screen part). I wasn't sure that the transition would work if the elements it affects are not in the component that it extends with `.sharedElements`. Maybe I'll try and see what happens.